src/loaders/OmeZarrLoader.ts

```diff
@@ -215,44 +103,62 @@ function convertChannel(channelData: zarr.TypedArray<zarr.NumberDataType>): Uint
   return u8;
 }
 
-type NumericZarrArray = zarr.Array<zarr.NumberDataType>;
+type NumericZarrArray = zarr.Array<zarr.NumberDataType, WrappedStore<RequestInit>>;
+
+export type ZarrLoaderFetchOptions = {
+  /** The max. number of requests the loader can issue at a time. Ignored if the constructor also receives a queue. */
+  concurrencyLimit?: number;
+  /**
+   * The max. number of *prefetch* requests the loader can issue at a time. Set lower than `concurrencyLimit` to ensure
+   * that prefetching leaves room in the queue for actual loads. Ignored if the constructor also receives a queue.
+   */
+  prefetchConcurrencyLimit?: number;
+  /**
+   * The max. number of chunks to prefetch outward in either direction. E.g. if a load requests chunks with z coords 3
+   * and 4 and `maxPrefetchDistance` in z is 2, the loader will prefetch similar chunks with z coords 1, 2, 5, and 6
+   * (or until it hits `maxPrefetchChunks`). Ordered TZYX.
+   */
+  maxPrefetchDistance: [number, number, number, number];
+  /** The max. number of total chunks that can be prefetched after any load. */
+  maxPrefetchChunks: number;
+};
 
-class OMEZarrLoader implements IVolumeLoader {
-  scaleLevels: NumericZarrArray[];
-  multiscaleMetadata: OMEMultiscale;
-  omeroMetadata: OmeroTransitionalMetadata;
-  axesTCZYX: [number, number, number, number, number];
+const DEFAULT_FETCH_OPTIONS = {
+  maxPrefetchDistance: [5, 5, 5, 5] as [number, number, number, number],
+  maxPrefetchChunks: 30,
```

Comment:
Idea/ Future suggestion: make this match the defaults explicitly passed in by the test app, but allow them to be tweaked at runtime using the advanced settings gui